CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.0
+
+This is a feature release that also contains bug fixes.
+
+* Added a new "v2" API that is based on the [Jookia](https://jolokia.org) library. Jookia
+  provides a superset of functionality found in the current "v1" metrics API. Refer to 'document/configureation.md'
+  for more information on setting up Jookia with Trapperkeeper. The [Jookia Documentation](https://jolokia.org/documentation.html)
+  has the specifics on the new metrics API.
+
+* Ensure the JSON request is fully parsed before closing the input stream of the request
```

Comment:
I know we haven't been doing it in this changelog, so, not the end of the world... but for this issue and the Jolokia one it'd  be nice to have a link to the Jira tickets.